### PR TITLE
RDAPI-24557 - Update kpsadmin documentation to include new scripting commands for table CRUD and recreate

### PR DESCRIPTION
**Pull request by dshielaxway**
Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

Update kpsadmin documentation to include new scripting commands for table CRUD and recreate

## Deploy preview link

https://deploy-preview-1963--axway-open-docs.netlify.app/docs/apim_policydev/apigw_kps/how_to_use_kpsadmin_command/#run-kpsadmin-operations-in-scriptable-command-mode

## Checklist for contributors

Before submitting this PR, please make sure:

* [ ] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [ ] You have signed the [Axway CLA](https://cla.axway.com/)
* [ ] You have verified the technical accuracy of your change
* [ ] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [ ] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [ ] You have verified that all status checks have passed

_Put an x in the boxes that apply. This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

File: content/en/docs/apim_policydev/apigw_kps/how_to_use_kpsadmin_command.md [hunk 1]
```diff
@@ -228,6 +235,50 @@ To show table details:
...
 
+### Create a new row
+
+To create a new row in a table:
+
+```
+./kpsadmin --username admin --password changeme --group "Staging" --name "Gateway1" --collection "My Collection" --table "My Table" create_row --datafile data.json
+```
+
+The data file contains a JOSN object representation of the row to be created. Autogenerated fields need not be included.
```
Comment (kmercierwhite) [inline content/en/docs/apim_policydev/apigw_kps/how_to_use_kpsadmin_command.md:246]: JOSN > JSON

File: content/en/docs/apim_policydev/apigw_kps/how_to_use_kpsadmin_command.md [hunk 2]
```diff
@@ -228,6 +235,50 @@ To show table details:
...
+
+### Re-create KPS table
+
+To re-create a KPS table:
+
+```
+./kpsadmin --username admin --password changeme --group "Staging" --name "Gateway1" --collection "My Collection" --table "My Table" recreate
+```
+
+{{< alert title="Caution" color="warning" >}}This command will drop and re-create the specified table with the loss of all data.{{< /alert >}}
```
Comment (kmercierwhite) [inline content/en/docs/apim_policydev/apigw_kps/how_to_use_kpsadmin_command.md:280]: This should probably be before the command, not after

File: content/en/docs/apim_policydev/apigw_kps/how_to_use_kpsadmin_command.md (hunk 1 above)
Comment (dshielaxway) [reply content/en/docs/apim_policydev/apigw_kps/how_to_use_kpsadmin_command.md]: Good spot, thanks!

File: content/en/docs/apim_policydev/apigw_kps/how_to_use_kpsadmin_command.md (hunk 2 above)
Comment (dshielaxway) [reply content/en/docs/apim_policydev/apigw_kps/how_to_use_kpsadmin_command.md]: Done

**Comment by sgoka-axway** [comment]
Looks good Dave!

**Comment by kmercierwhite** [comment]
Thanks Dave, looks good to me!

**Review by maherb4 — APPROVED** [review]

**Review by kmercierwhite — APPROVED** [review]

**Pull request merged**